library/src/lib/calendar/calendar2/calendar2-views/calendar2-month-view/calendar2-month-view.component.html

```diff
@@ -1,3 +1,22 @@
-<p>
-  calendar2-month-view works!
-</p>
+<div class="fd-calendar__months">
+    <ul class="fd-calendar__list">
+        <li
+            class="fd-calendar__item"
+            *ngFor="let month of monthNames; let i = index"
+            [ngClass]="
+                (i + 1 === currentMonth ? ' fd-calendar__item--current' : '') +
+                (i + 1 === monthSelected ? ' is-selected' : '')
+            "
+            (click)="selectMonth(i, $event)"
+        >
+            <span
```

Comment:
is this standard from fd? this is bad for a11y, a span with role button should just be a button.

also this needs an aria-label of the month name localized

Reply:
![Screen Shot 2019-07-12 at 9 47 26 AM](https://user-images.githubusercontent.com/39598672/61132823-297cd600-a48a-11e9-876c-be712e80b0f1.png)

Reply:
https://sap.github.io/fundamental-styles/components/calendar.html

Reply:
Ok well we should change that in FD to buttons probably lol